ark/utils/example_dataset_test.py

```diff
@@ -32,7 +32,7 @@ def dataset_download(request) -> Iterator[ExampleDataset]:
     example_dataset: ExampleDataset = ExampleDataset(
         dataset=request.param,
         cache_dir=None,
-        revision="main"
+        revision="1cab29d817d03b924edd32e6b3e2e13e61160c57"
```

Comment:
Once this gets changed back we're good to go

Reply:
@ngreenwald I make sure that happens right before I merge the PR into `main`. So if you approve this PR I can then update HuggingFace, push a final change on this PR with `revision='main'`, then immediately squash and merge the PR into `main` with minimal downtime.

If we're OK with updating HuggingFace right now with the `revision='main'` commit, I can do that as well, then re-request the review.